### PR TITLE
DOC: Added simple example for registering instruments within pysat

### PR DESCRIPTION
**Pull request by rstoneback**
# Description

Addresses #837 

Adds a simple example for registering instrument support within pysat in tutorial_basics.

## Type of change

Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide
instructions so we can reproduce. Please also list any relevant details for
your test configuration

- Built documentation locally and inspected visually

**Test Configuration**:
* Operating system: MacOS
* Version number: Python 3.8.3
* Any details about your local setup that are relevant

# Checklist:

- [ ] Make sure you are merging into the ``develop`` (not ``main``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes

If this is a release PR, replace the first item of the above checklist with the release
checklist on the wiki: https://github.com/pysat/pysat/wiki/Checklist-for-Release

### CODE REVIEW

File: docs/tutorial/tutorial_basics.rst [hunk 1]
```diff
@@ -80,7 +80,23 @@ name instead of the platform and name because non-registered Instruments are
 instantiated using the ``inst_module`` keyword instead of the ``platform`` and
 ``name`` keywords (jump to the :ref:`instantiation` section below for more
 information).  To use the ``platform`` and ``name`` keywords, the instrument
-must be registered.  To display the registered instruments, no input is needed.
+must be registered. To register a group of instruments within a module,
```
Comment (aburrell) [inline docs/tutorial/tutorial_basics.rst:83]: recommend including a link to the Register API here as well.

File: docs/tutorial/tutorial_basics.rst [hunk 2]
```diff
@@ -80,7 +80,24 @@ name instead of the platform and name because non-registered Instruments are
...
+
```
Comment (aburrell) [inline docs/tutorial/tutorial_basics.rst:84]:
```suggestion
follow the example below.

```

File: docs/tutorial/tutorial_basics.rst [hunk 3]
```diff
@@ -80,7 +80,24 @@ name instead of the platform and name because non-registered Instruments are
...
+.. code:: python
+
+    import pysat
+    import pysatMadrigal as pysatMad
+    pysat.utils.registry.register_by_module(pysatMad.instruments)
+
+or to register a more select group of instruments,
```
Comment (aburrell) [inline docs/tutorial/tutorial_basics.rst:91]:
```suggestion
You may also register a more select group of instruments.
```

File: docs/tutorial/tutorial_basics.rst [hunk 4]
```diff
@@ -80,7 +80,24 @@ name instead of the platform and name because non-registered Instruments are
...
+
+.. code:: python
+
+    import pysat
+    pysat.utils.registry.register(['pysatMadrigal.instruments.dmsp_ivm',
+                                   'pysatMadrigal.instruments.jro_isr'])
+
+see :py:mod:`pysat.utils.registry` for more. To display the registered
```
Comment (aburrell) [inline docs/tutorial/tutorial_basics.rst:99]:
```suggestion
See :py:mod:`pysat.utils.registry` for more details. To display the registered
```

**Review by aburrell — COMMENTED** [review]
Made some wording suggestions so the examples don't create a run-on sentence.  Also directly committed a change to the python reST styling.

File: docs/tutorial/tutorial_basics.rst (hunk 4 above)
Comment (rstoneback) [reply docs/tutorial/tutorial_basics.rst]: Thanks for the correction. I did copy from a reference elsewhere in the docs to try and keep within standards but I put have missed a difference in application from the source.

Comment (aburrell) [reply docs/tutorial/tutorial_basics.rst]: Or maybe you copied from something I missed, or something older?  In any case, it's easy enough to catch the things we add in.

**Review by aburrell — APPROVED** [review]

**Comment by rstoneback** [comment]
Thanks for the improvements @aburrell!

**Pull request merged**